test/functional/wallet_multiwallet.py

```diff
@@ -303,11 +303,8 @@ def wallet_file(name):
 
         # Fail to load duplicate wallets
         path = os.path.join(self.options.tmpdir, "node0", "regtest", "wallets", "w1", "wallet.dat")
```

Comment:
```suggestion
```
A small nit, I think this line is not really needed, since the variable **path** is not used anymore with the new error message.